.backportrc.json

```diff
@@ -2,14 +2,17 @@
   "repoOwner": "microsoft",
   "repoName": "ccf",
   "targetBranchChoices": ["release/2.x"],
-
   "branchLabelMapping": {
     "^(.+)-todo$": "release/$1"
   },
   "autoMerge": true,
   "autoMergeMethod": "squash",
   "prTitle": "[{targetBranch}] Cherry pick: {commitMessages}",
-  "prDescription": "Backports the following commits to {targetBranch}:\n{commitMessages}",
+  "prDescription": "Backports the following commits to `{targetBranch}`:\n{commitMessages}",
   "sourcePRLabels": ["backported"],
-  "targetPRLabels": ["backport"]
+  "targetPRLabels": ["backport"],
+  "autoAssign": false,
+  "publishStatusCommentOnSuccess": false,
+  "publishStatusCommentOnFailure": false,
+  "publishStatusCommentOnAbort": false
```

Comment:
I don't have a problem with verbose comments to track what a bot did, and seeing when it failed seems helpful.